src/shared/loadPseudoData.c

```diff
@@ -0,0 +1,29 @@
+#include "loadPseudoData.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int loadData(const char *input_file, const char *output_file) {
+    FILE *ifp, *ofp;
+
+    if ((ifp = fopen(input_file, "r")) == NULL) {
+        printf("Error : Unable to open %s for reading\n", input_file);
+        exit(EXIT_FAILURE);
+    }
+    if ((ofp = fopen(output_file, "w+")) == NULL) {
+        printf("Error : Unable to open %s for reading\n", output_file);
```

Comment:
Please write to *stderr* when you have an error so its more visible in the console.
[how to print to stderr](https://stackoverflow.com/questions/39002052/how-i-can-print-to-stderr-in-c)